vividus-tests/src/main/resources/story/integration/Load ExamplesTable from local file.story

```diff
@@ -0,0 +1,9 @@
+Meta:
+    @epic vividus-bdd-engine
+    @feature variables
+
+
+Scenario: Should use Examples Table from temporary local file
+Then `<name>` is equal to `common`
+Examples:
+file:///${examples-table-temporary-file}
```

Comment:
Could you please elaborate why `///`?

Reply:
https://datatracker.ietf.org/doc/html/rfc8089#appendix-A